bindings/rust/bench/scripts/install-aws-lc.sh

```diff
@@ -3,34 +3,43 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+# Usage: ./install-aws-lc.sh
 # Sets up bench crate to use aws-lc for either s2n-tls or rustls if desired
-# To run with aws-lc, use any cargo command with:
+
+# To run benches with aws-lc, use any cargo command with:
 # --config aws-lc-config/s2n.toml
 # --config aws-lc-config/rustls.toml
 # or both
 
+# How Rustls with aws-lc-rs works:
+# Clones aws-lc-rs, changes its name to ring with a compatible version number, and
+# patches it into Rustls
+
+# How s2n-tls with aws-lc works:
+# Builds s2n-tls static lib with AWS-LC interned, required to avoid symbol collisions with OpenSSL
+# Checks for libs2n.a at target/s2n-tls-build/lib/libs2n.a
+# Checks for libcrypto.a at target/aws-lc/install/lib/libcrypto.a
+
 set -e 
 
 # go to bench directory
-pushd "$(dirname "$0")" > /dev/null
+pushd "$(dirname "$0")"/.. > /dev/null
 bench_dir="$(pwd)"
 
 
 
 # ----- rustls -----
 
-# clone rustls to bench/target/rustls and checkout compatible version
-rm -rf target/rustls
-git clone https://github.com/rustls/rustls target/rustls
-cd target/rustls
-git checkout 'v/0.21.5'
-
-# go to dir with rustls crate
-cd rustls
-rustls_dir="$(pwd)"
+# clone aws-lc-rs to target
+rm -rf target/aws-lc-rs
+git clone https://github.com/aws/aws-lc-rs target/aws-lc-rs
+cd target/aws-lc-rs/aws-lc-rs
+git submodule init
+git submodule update
 
-# change rustls to use aws-lc-rs
-sed -i 's|ring = .*|ring = { package = "aws-lc-rs" }|' Cargo.toml
+# change aws-lc-rs to look like API compatible ring (name and version)
+# assumes name and version are in the first 5 lines of the Cargo.toml and replaces them
+sed -i '1,5s|name = .*|name = "ring"| ; 1,5s|version = .*|version = "0.16.20"|' Cargo.toml
```

Comment:
this is a bit more brittle. why the first 5 lines?

Reply:
There was a way to do it to to only change the first occurrence (because other occurrences of `version = .*` that shouldn't change happen in the dependencies), and I didn't figure out how to do it. However, I realized I just needed to used `/` as the delimeter for `sed` instead of `|` for the pattern matching and it just works. Just changed it and updated the comment with it.